tensorflow/python/ops/image_ops_impl.py

```diff
@@ -5536,6 +5536,20 @@ def generate_bounding_box_proposals(scores,
                                     name=None):
   """Generate bounding box proposals from encoded bounding boxes.
 
+  Args:
+    scores: A 4-D float tensor of shape `[num_images, height, width, num_achors]` 
```

Comment:
tensor --> `Tensor`

Reply:
I guess the `Tensor` was needed everywhere in the args, so changed to accordingly.